src/Domains/topography.jl

```diff
@@ -0,0 +1,29 @@
+abstract type AbstractTopography end
+
+"""
+  CanonicalSurface <: AbstractTopography
```

Comment:
Is there a reason why we have both CanonicalSurface and NoTopgraphy?

Reply:
Nope - changing names to only support `WarpedSurface` and `CanonicalSurface`